Automattic/wp-calypso
# Jetpack Manage: Upsell lightbox component - adding tests to confirm expected behaviour

## Proposed Changes

* This PR adds tests to the upsell components in Jetpack Manage.

Project thread: pf36In-9j-p2

## Testing Instructions

* The tests should run in the PR, but you can also run `yarn run test-client client/components/jetpack/upsell-product-card/test/` and change the results in any of the test code to confirm they're running as expected in this PR.

## Pre-merge Checklist
